rules/app.bzl

```diff
@@ -67,7 +83,6 @@ def ios_application(name, apple_library = apple_library, infoplists_by_build_set
     )
 
     application_kwargs["launch_storyboard"] = application_kwargs.pop("launch_storyboard", library.launch_screen_storyboard_name)
-    application_kwargs["families"] = application_kwargs.pop("families", ["iphone", "ipad"])
```

Comment:
Not sure why we decided to default this, but agree it's probably best to not default here and instead make sure the user provides it.

This will be a breaking change though so not sure if its worth it? Let me know

Reply:
I also don't know why we default-ed here, since this is required on the `rules_apple` side I think it makes sense to default to empty and allow consumers to configure this properly

I'll tag @justinseanmartin and @congt here since they might have context on older versions of this file and raise any concerns with this breaking change in favour of a more reasonable default

That said, I think it's a small change on users BUILD files, so I'd only consider not doing it if there's a major concern here

Reply:
I'm ok with not reverting it then 👍, +1 to making it visible in the docs

Reply:
@mattrobmattrob so just to confirm the preference would be to keep the two families as default everywhere or only for apps?

Reply:
I suppose I'd prefer the default (assuming we keep it) is the same anywhere it's applicable as a param to the underlying rules.

Reply:
I'll update to move that to the macro args so its documented in all places it was previously used 👍🏼